csob/resources/payment/init.py

```diff
@@ -0,0 +1,100 @@
+import json
+from typing import Optional, List
+
+from csob.resources.payment import PaymentCSOBResource
+
+
+class PaymentInit(PaymentCSOBResource):
+    url = 'payment/init'
+    request_signature = (
+        'merchantId', 'orderNo', 'dttm', 'payOperation', 'payMethod', 'totalAmount', 'currency', 'closePayment',
+        'returnUrl', 'returnMethod', 'cart', 'description', 'merchantData', 'customerId', 'language', 'ttlSec',
+        'logoVersion', 'colorSchemeVersion')
+    optional_request_signature = ('merchantData', 'customerId', 'ttlSec', 'logoVersion', 'colorSchemeVersion')
+
+    def post(self, key, order_number: str, pay_operation: str, pay_method: str, total_amount: int, currency: str,
```

Comment:
to je hrozná metoda. Nenapadá tě jak to přepsat, aby validace byla per hodnota?, vidím tam spoustu opakován